build.py

```diff
@@ -641,7 +664,7 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
-class Project_graphene(GitRepo, Project):
+class Project_graphene(GitRepo, Meson):
     def __init__(self):
         Project.__init__(self,

```

Comment:
Yes, my mistake. The result is the same (Meson __init__ calls the project __init__) but is correct to call the Meson one.